src/main/java/smartthings/konsumer/filters/RetryMessageFilter.java

```
@@ -0,0 +1,39 @@
+package smartthings.konsumer.filters;
+
+import kafka.message.MessageAndMetadata;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import smartthings.konsumer.filterchain.MessageContext;
+import smartthings.konsumer.filterchain.MessageFilter;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+@ThreadSafe
+public class RetryMessageFilter implements MessageFilter {
+
+	private final static Logger log = LoggerFactory.getLogger(RetryMessageFilter.class);
+
+	private final int retryCount;
+
+	public RetryMessageFilter(int retryCount) {
```

Would it make sense to add an option to just swallow the exception?